test/Sentry.Profiling.Tests/SamplingTransactionProfilerTests.cs

```diff
@@ -231,6 +248,67 @@ async Task VerifyAsync(HttpRequestMessage message)
         }
     }
 
+    [SkippableFact]
+    public async Task Profiler_ThrowingOnSessionStartup_DoesntBreakSentryInit()
+    {
+#if DEBUG
+        SampleProfilerSession.ThrowOnNextStartupForTests = true;
+#else
+        Skip.If(true, "This test only works in DEBUG mode.");
```

Comment:
Mentioned in a different comment but also means we don't run the test in CI

Reply:
changed to run all the time. the idea was to only have the code present when running in debug mode..